### PR TITLE
fix accessibilità headings e tabs

### PR DESCRIPTION
**Pull request by Fladiir**
## Description
fix accessibilità headings e tabs
Fix pull request 148

### CODE REVIEW

**Review by lollomag — APPROVED** [review]

File: contents/faq-page/faq.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+name: FAQ
```
Comment (sebbalex) [inline contents/faq-page/faq.yml:1]: as for Homepage should we use the syntax "Page name - PA digitale 2026"

File: contents/opportunity-page/opportunity.yml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+name: Le misure
```
Comment (sebbalex) [inline contents/opportunity-page/opportunity.yml:1]: same here

File: contents/support-page/support.yml [hunk 3]
```diff
@@ -1,3 +1,4 @@
+name: Supporto
```
Comment (sebbalex) [inline contents/support-page/support.yml:1]: same here

**Review by sebbalex — COMMENTED** [review]
A new dependency was added, I expect the `yarn.lock` file committed here.

**Review by sebbalex — APPROVED** [review]

**Pull request merged**